### PR TITLE
rails back-end api

### PR DESCRIPTION
**Pull request by anas599**
In this PR I added the bakend API using ruby on rails
and used it in front end with react [Here](https://github.com/anas599/hello-react-front-end/pull/1)

- Create a new Rails API app called 'hello-rails-back-end'
- Postgres database set up
- Create a table for storing messages
- Create an API endpoint that selects a random greeting
- Add a controller with an action and rails route for random greetings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+</details>
+<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/anas599/My-Rails-React/issues).
```
Comment (Munsa1) [inline README.md:145]: - [ ] It would be nice if the issues link would point to the issues page for the backend repository. Kindly fix this

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @anas599, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Neat Descriptive ReadMe's :heavy_check_mark: 
Descriptive Pr's :heavy_check_mark: 
The Two Apps Work Well Together :heavy_check_mark: 
Highly Clean Codebase :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Munsa1) [reply README.md]: Also, check the front-end Pr for a review

**Review by OyePriscilla — APPROVED** [review]
Hi @anas599 👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Implemented API endpoint for returning a random greeting that was created ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**